configs/tcp_bidir_flows.json

```diff
@@ -9,6 +9,13 @@
       "name": "rx"
     }
```

Comment:
This file needs to be fixed for str vs int (since I see TCP ports are all strings)

Reply:
This was made because of Athena.